src/OSPSuite.Infrastructure.Import/Core/ColumnInfo.cs

```diff
@@ -29,11 +29,17 @@ public string Name
       ///    Column name of related column. The related column must have an auxiliary type as meta data category.
       /// </summary>
       public string RelatedColumnOf { get; set; }
-      public IList<DimensionInfo> DimensionInfos { get; private set; }
+
+      /// <summary>
+      ///    List of possible supported dimensions.This List may be empty if all dimensions are supported
```

Comment:
I am not sure if this is true or not. Or should the caller add all dimensions?

Reply:
I am not sure either. But, is there any case where we have all dimensions?

Reply:
yes. MoBi adds everything for instance in a loop